Gems/Atom/RHI/Code/Include/Atom/RHI/CpuProfilerImpl.h

```diff
@@ -129,6 +131,11 @@ namespace AZ
             AZStd::vector<RHI::Ptr<CpuTimingLocalStorage>, AZ::OSStdAllocator> m_registeredThreads;
             AZStd::mutex m_threadRegisterMutex;
 
+            // Storage for runtime GroupRegionNames
+            AZStd::unordered_set<CachedTimeRegion::GroupRegionName, CachedTimeRegion::GroupRegionName::Hash> m_dynamicGroupRegionNameSet;
```

Comment:
Can you add a comment clarifying how m_dynamicGroupRegionNameSet and m_dynamicRegionNameSet will be used.